students/arun_nalla/session04/Lesson4_Mailroom_Part2.py

```diff
@@ -0,0 +1,74 @@
+#!usr/bin/env python
+
+donors_list = [('Aron King', [1, 2]), ('Becky Patty', [3000, 1000]),
+               ('Charlie Lee', [666, 222]), ('Thomas Dave', [202, 90, 450]), ('Nancy Crow', [100, 200, 300])]
+
+donors_dict = {x: y for x, y in donors_list}
```

Comment:
Minor: FYI, `donors_dict` can be directly initialized as below, not through another list (`donors_list`):
```
donors_dict = { 'Aron King': [1,2], 'Becky Patty': [3000, 1000], ... }
```